src/server/game/Server/Packets/AchievementPackets.h

```diff
@@ -38,7 +38,7 @@ namespace WorldPackets
         {
             uint32 Id = 0;
             uint64 Quantity = 0;
-            ObjectGuid Player;
+            ObjectGuid Owner;
```

Comment:
the original name comes from debug version of 6.0.1 client (account wide achievements were supported already) so i want the name unchanged (its Player)